DuckDuckGo/TabViewController.swift

```diff
@@ -74,7 +74,11 @@ class TabViewController: UIViewController {
         }
     }
     
-    weak var delegate: TabDelegate?
+    weak var delegate: TabDelegate? {
+        didSet {
+            assert(delegate != nil)
```

Comment:
This is getting triggered when using fire button, as `prepareForDataClearing` sets delegate to nil